pkgs/development/python-modules/humanize/default.nix

```
@@ -22,7 +22,7 @@ buildPythonPackage rec {
     owner = "python-humanize";
     repo = pname;
```

`repo` shouldn't be determined by `pname`.

```suggestion
    repo = "humanize";
```

---

Done.